src/routes/index.js

```diff
@@ -47,16 +54,18 @@ apiRouter.get("/admin/remind_confirm", adminAuthMiddleware, application.remindCo
 apiRouter.get("/admin/remind_apply", adminAuthMiddleware, application.remindApply);
 apiRouter.post("/admin/schedule/create", adminAuthMiddleware, schedule.create);
 apiRouter.get("/admin/schedule/read", adminAuthMiddleware, schedule.read);
-apiRouter.put("/admin/schedule/update", adminAuthMiddleware, schedule.update)
-apiRouter.delete("/admin/schedule/remove", adminAuthMiddleware, schedule.remove);
-
+apiRouter.put("/admin/schedule/update", adminAuthMiddleware, schedule.update);
+apiRouter.delete(
+  "/admin/schedule/remove",
```

Comment:
Also why did you change the format of these for only some of the routes?

Reply:
why did I do? I can't tell what you referring to?